TADashboard/src/main/java/page_objects/DataProfilesDisplayFieldsPage.java

```diff
@@ -0,0 +1,14 @@
+package page_objects;
+
+import com.logigear.control.common.imp.Label;
+
+import java.util.List;
+
+public class DataProfilesDisplayFieldsPage {
+    private Label lblDisplayField = new Label("css=table[id='profilesettings'] tbody label");
+
+    public List<String> getTextOfCheckBox() {
```

Comment:
it is display field, but why your method is getting text of checkbox?

Reply:
I fixed it. Please check agian